exhibitor-appliance.yaml

```diff
@@ -69,10 +71,18 @@ Resources:
       - InstancePort: 8181
         LoadBalancerPort: 80
         Protocol: TCP
-      LoadBalancerName: "exhibitor-{{Arguments.version}}"
+      - InstancePort: 2181
+        LoadBalancerPort: 2181
+        Protocol: TCP
+      - InstancePort: 2888
```

Comment:
ZooKeeper client needs only port 2181 (2888, 3888 doesn't make sense to load balance)
Actually any application which relies on ZooKeeper should communicate with nodes directly because it takes some time for load balancer to bring node into healthy state and usually application doesn't have this time. It's much faster to go through list of nodes then wait until load balancer would be updated....

Yes, it could be useful to have 2181 in load balancer, but only for testing purposes. 8181 is really needed, because through it we can get list of all cluster nodes.